dashboard/src/features/orgs/projects/services/components/ServiceForm/components/ImageFormSection/ImageFormSection.tsx

```diff
@@ -0,0 +1,179 @@
+import { Box } from '@/components/ui/v2/Box';
+import { ArrowSquareOutIcon } from '@/components/ui/v2/icons/ArrowSquareOutIcon';
+import { InfoIcon } from '@/components/ui/v2/icons/InfoIcon';
+import { Input } from '@/components/ui/v2/Input';
+import { Link } from '@/components/ui/v2/Link';
+import { Text } from '@/components/ui/v2/Text';
+import { Tooltip } from '@/components/ui/v2/Tooltip';
+import { Label } from '@/components/ui/v3/label';
+import { RadioGroup, RadioGroupItem } from '@/components/ui/v3/radio-group';
+import { useIsPlatform } from '@/features/orgs/projects/common/hooks/useIsPlatform';
+import { InfoCard } from '@/features/projects/overview/components/InfoCard';
+import type { ServiceFormValues } from '@/features/services/components/ServiceForm/ServiceFormTypes';
+import { useFormContext } from 'react-hook-form';
+
+interface ImageFormSectionProps {
+  serviceID?: string;
+  onImageTypeChange?: (value: 'public' | 'private' | 'nhost') => void;
+  privateRegistryImage?: string;
+  imageType?: 'public' | 'private' | 'nhost';
+}
+
+export default function ImageFormSection({
+  privateRegistryImage,
+  imageType,
+  onImageTypeChange,
+  serviceID,
+}: ImageFormSectionProps) {
+  const {
+    register,
+    formState: { errors },
+    watch,
+  } = useFormContext<ServiceFormValues>();
+
+  const image = watch('image');
+
+  const isPlatform = useIsPlatform();
+
+  return (
+    <Box className="space-y-4 rounded border-1 p-4">
+      <Box className="flex flex-row items-center space-x-2">
+        <Text variant="h4" className="font-semibold">
+          Image
+        </Text>
+        <Tooltip
+          title={
+            <span>
+              Image to use, it can be hosted on any public registry or it can
```

Comment:
This text seems to be a bit outdated, maybe the tooltip in image can just be removed